.changeset/crisp-beds-bathe.md

```
@@ -0,0 +1,8 @@
+---
+'@mastra/libsql': minor
+'@mastra/pg': minor
+'@mastra/core': patch
+'mastra': patch
+---
+
+Added storage for live-scoring sampling decision records (a new mastra_scoring_decisions table) so scoring coverage can be computed from sampled and declined decisions.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)